viewing_party/party.py

```diff
@@ -1,23 +1,219 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+def create_movie(movie_title, genre, rating):
+    my_movie = {
+        "title" : movie_title,
+        "genre" : genre,
+        "rating" : rating,
+    }
+    for keys in my_movie:
+        if my_movie[keys] == False:

```

Comment:
Following up from the earlier comment, I think the issue here is that you're comparing the parameter values to `False`, but I think you meant to compare them all to `None` or if you want to check if they're "falsy", you could do 
```python
if not movie_title or not genre or not rating:
    return None
# otherwise return the dictionary
return {
    "title" : movie_title,
    "genre" : genre,
    "rating" : rating
}
```